### PR TITLE
0.0.13

### PR DESCRIPTION
**Pull request by jasongrout**
Update.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 package:
   name: jupyterlab
   version: {{ version }}
 
 source:
     fn: jupyterlab-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/d6/46/dc0752f26c67b91ba71fecc230f107e95b8793ad51e679f02158a5de2d0e/jupyterlab-{{ version }}.tar.gz
-    md5: d93e2774721f1d85a657d548b03071da
+    url: https://pypi.python.org/packages/87/5f/05f08de58c835012a119b83557e04d66af88b238141b329059e903b78566/jupyterlab-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipe/meta.yaml]: Please change the URL to

```
https://pypi.io/packages/source/j/jupyterlab/jupyterlab-{{ version }}.tar.gz
```

Comment (jakirkham) [inline recipe/meta.yaml]: This should make it more maintainable over the long run.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+    md5: 33a730a8880023633ffb36f2bbd34c5
```
Comment (jakirkham) [inline recipe/meta.yaml]: Also, think we lost a `c` from the end.

File: recipe/meta.yaml (hunk 1 above)
Comment (jasongrout) [inline recipe/meta.yaml]: Thanks! I've wondered what the right non-version-specific url was!

**Pull request merged**